cmake/compiler/gcc/target.cmake

```diff
@@ -49,11 +49,7 @@ elseif("${ARCH}" STREQUAL "arc")
     -mcpu=${GCC_M_CPU}
     )
 elseif("${ARCH}" STREQUAL "riscv")
-  if(CONFIG_64BIT)
-    list(APPEND TOOLCHAIN_C_FLAGS -mabi=lp64 -march=rv64imac -mcmodel=medany)
-  else()
-    list(APPEND TOOLCHAIN_C_FLAGS -mabi=ilp32 -march=rv32ima)
-  endif()
+  include(${ZEPHYR_BASE}/cmake/compiler/gcc/target_riscv.cmake)
```

Comment:
include(${CMAKE_CURRENT_LIST_DIR}/target_riscv.cmake)

Reply:
I'm AFK until Monday. I'll make this update then

Reply:
Complete